### DISCUSSION TITLE
File column resizing

### DISCUSSION THREAD

**Discussion by musictrivianut** [comictagger/comictagger#493] [Q&A] [answered]
When using the GUI (v1.6.0a5.dev38, if it matters) and dragging files over to it, the File column seems to resize randomly.  If there is a pattern to it, I can't figure out what it is.  Does seem to vary though.

Is there any way to get it to stay where I set it?  Because it is definitely not resizing to where I can see the entire filename and that is what I would like to be able to do.

Even if not, my thanks to the developer(s).  Great bit of software!

**Comment by lordwelch** [comment, marked as answer]
It does resize the columns to fit the contents, it however limits the size to 250 pixels.

**Comment by protofolius** [comment]
I add some files . Drag the first column to the right so I can read the names properly .
Then I add 1 more file ... it shortens the column again and I need to manually drag it again so I can read the names properly .
Add 1 more file ? Column again shortens .....

Frustrating .
